.github/workflows/build.yml

```diff
@@ -85,6 +85,9 @@ jobs:
         run: |
           npm whoami
           node ./scripts/npmPublish.js --provenance
+          npm run -w demo update:stackblitzPkgs ${{ inputs.version }}
+          npm run build -w demo
+          git commit -a -m "feat: update stackblitz packages to ${{ inputs.version }}"
```

Comment:
You should move the creation of the version tag after this command if you want this new commit to be pushed.